Description: Add setting for gtk overlay scrollbars.
Author: miketwebster@gmail.com
Origin: upstream, https://github.com/linuxmint/cinnamon-desktop/commit/9076d6217ae8a7a5b427c29f95c183b48ee0d751
Bug: https://github.com/linuxmint/Cinnamon/issues/5306
Last-Update: 2016-05-18
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: cinnamon-desktop/schemas/org.cinnamon.desktop.interface.gschema.xml.in.in
===================================================================
--- cinnamon-desktop.orig/schemas/org.cinnamon.desktop.interface.gschema.xml.in.in	2015-06-09 19:03:57.457766655 +0200
+++ cinnamon-desktop/schemas/org.cinnamon.desktop.interface.gschema.xml.in.in	2016-05-18 21:54:00.810102759 +0200
@@ -256,5 +256,9 @@
         presses the Alt key.
       </_description>
     </key>
+    <key name="gtk-overlay-scrollbars" type="b">
+      <default>true</default>
+      <_summary>Enable or disable gtk overlay scrollbars</_summary>
+    </key>
   </schema>
 </schemalist>
